sql/core/benchmarks/DataSourceReadBenchmark-results.txt

```diff
@@ -2,430 +2,459 @@
-Partition column - Parquet MR: DataPageV2                   1429           1437          11         11.0          90.8      18.4X
-Partition column - ORC Vectorized                             63             68           4        250.2           4.0     418.1X
-Partition column - ORC MR                                   1595           1605          13          9.9         101.4      16.5X
-Both columns - CSV                                         25899          26018         168          0.6        1646.6       1.0X
-Both columns - Json                                        14667          14685          26          1.1         932.5       1.8X
-Both columns - Parquet Vectorized: DataPageV1                288            295          10         54.7          18.3      91.4X
-Both columns - Parquet Vectorized: DataPageV2                621            624           3         25.3          39.5      42.3X
-Both columns - Parquet MR: DataPageV1                       2905           2945          57          5.4         184.7       9.0X
-Both columns - Parquet MR: DataPageV2                       2772           2790          25          5.7         176.2       9.5X
-Both columns - ORC Vectorized                                401            405           5         39.3          25.5      65.6X
-Both columns - ORC MR                                       2601           2610          12          6.0         165.4      10.1X
+Data column - CSV                                          21677          21683           8          0.7        1378.2       1.0X
+Data column - Json                                         15095          15156          85          1.0         959.7       1.4X
+Data column - Parquet Vectorized: DataPageV1                 171            176           8         92.0          10.9     126.8X
+Data column - Parquet Vectorized: DataPageV2                 413            416           5         38.1          26.2      52.5X
+Data column - Parquet MR: DataPageV1                        2501           2511          13          6.3         159.0       8.7X
+Data column - Parquet MR: DataPageV2                        2356           2370          20          6.7         149.8       9.2X
+Data column - ORC Vectorized                                 279            283           4         56.3          17.8      77.6X
+Data column - ORC MR                                        2012           2016           6          7.8         127.9      10.8X
+Partition column - CSV                                      6557           6595          53          2.4         416.9       3.3X
+Partition column - Json                                     9875           9890          21          1.6         627.8       2.2X
+Partition column - Parquet Vectorized: DataPageV1             38             43          10        410.9           2.4     566.4X
```

Comment:
This scene is also a little improved, and the proportion is more than the orc that has not been changed